_episodes/02-starting-with-data.md

```diff
@@ -18,7 +18,11 @@ objectives:
     - "Perform basic mathematical operations and summary statistics on data in a Pandas DataFrame."
     - "Create simple plots."
 keypoints:
-    - "FIXME"
+    - "DataFrames objets are data structure."
```

Comment:
1. objets
2. This keypoint is meaningless

Reply:
Remove this please